### ISSUE TITLE
Too perfect?

### ISSUE THREAD

**Issue by BUTTyPOO** [BUTTyPOO/EasterGame#1]
I just feel like its too perfect and it makes the other REPOs look like trash. If This gets removed u know why

**Comment by jfallz** [comment]
I agree, but please don't post topics already answered in the readme. Thread closed.
